packages/widgetbook/test/src/addons/custom_theme_addon_test.dart

```diff
@@ -0,0 +1,210 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:widgetbook/src/addons/addons.dart';
+import 'package:widgetbook/src/addons/theme_addon/theme_selection_provider.dart';
+import 'package:widgetbook/src/workbench/renderer.dart';
+
+import 'utils/custom_app_theme.dart';
+import 'utils/widget_wrapper.dart';
+
+void main() {
+  late Renderer sut;
+  final rendererKey = GlobalKey();
+
+  setUpAll(() {
```

Comment:
I think we should rarely use `setUpAll` and use `setUp` instead since obtaining a fresh instance will guarantee that there are no side effects from other tests